packages/cli/lib/zeroYaml/definitions.ts

```diff
@@ -119,6 +142,31 @@ export function buildSync({
         }
     }
 
+    // Validation
+    // TODO: We should probably share this with the backend and have a single zod validation
+    const interval = getInterval(params.frequency, new Date());
+    if (interval instanceof Error) {
```

Comment:
Why do we return an error getInterval instead of throwing it.

Reply:
legacy things, before we introduced Result types